src/components/PlayerTab.vue

```diff
@@ -74,30 +101,39 @@
         :min="0"
         hide-details
       >
+        <template v-slot:append>
+          <div class="slider_append">
+            {{
+              global.freezing.m_CurrentFreezingProxy
+                ? round(global.freezing.m_CurrentFreezingProxy)
+                : 0
+            }}
+            <span class="suffix">%</span>
+          </div>
+        </template>
       </v-slider>
     </template>
-
   </div>
-
 </template>
 
 <script setup lang="ts">
-import store from '../store'
-import { computed, reactive, ref, watch } from 'vue';
-import { freemem } from 'os';
-import { glob } from 'glob';
+import store from '../store';
+import { computed } from 'vue';
 
 const global = computed(() => store.global);
 const condition = computed(() => store.global?.condition);
 
-
+function round(value: number) {
+  return Math.round(value);
+}
 </script>
 
 <style lang="scss" scoped>
 .container {
   margin: 2em;
   display: grid;
   grid-template-columns: max-content auto;
+  grid-template-rows: repeat(7, 50px); 
```

Comment:
You can remove this if you start using scoped CSS. Right now the map tab CSS is bleeding to this component because it's not scoped.